crates/bevy_ecs/src/system/exclusive_system.rs

```diff
@@ -9,8 +9,38 @@ pub trait ExclusiveSystem: Send + Sync + 'static {
 
     fn id(&self) -> SystemId;
 
+    /// Runs the exclusive system in the world.
+    ///
+    /// Use [`run_direct`] instead if you are manually running a system outside of a schedule
+
     fn run(&mut self, world: &mut World);
```

Comment:
Does this imply that run has an additional safety requirement that `self` is initialised for this world?

If that is so, then it should be `unsafe`

Reply:
It's not *unsafe* in the Rust sense; it merely panics with an opaque error message.